modules/template/src/main/twirl/scaladex/view/project/artifact.scala.html

```diff
@@ -52,6 +52,9 @@ <h3>
               @info("Group ID") { @artifact.groupId }
               @info("Artifact ID") { @artifact.artifactId }
               @info("Version") { @artifact.version }
+              @artifact.versionScheme.map { scheme =>
+                @info("Version Scheme")(Html(scheme))
```

Comment:
I suggest to make a link to https://www.scala-sbt.org/1.x/docs/Publishing.html#Version+scheme, for people who don't know what the version scheme is.

Reply:
done